### ISSUE TITLE
Hi,I have a question

### ISSUE THREAD

**Issue by YueZhangX** [milesial/Pytorch-UNet#200]
Thanks for your code sharing, it helps me a lot!
I have a question why you use a sigmoid function after the prediction and before the threshold, like:

`with torch.no_grad():
            pre_img = net(img)

            if net.n_classes > 1:
                probs = F.softmax(pre_img, dim=1)
            else:
                probs = torch.sigmoid(pre_img)
            
            probs = probs.squeeze(0)
            probs = trans(probs.cpu())
            full_mask = probs.squeeze().cpu().numpy()
        
        pre_img = full_mask > threshold`

I don't know the reason why you use the sigmoid.
Thanks!!!

**Comment by milesial** [comment]
When the classes are binary and we have only one output layer, we apply sigmoid to extract probabilities from raw logits.

**Comment by YueZhangX** [comment]
> When the classes are binary and we have only one output layer, we apply sigmoid to extract probabilities from raw logits.

Thanks！！
